app/constants/claim.js

```diff
@@ -5,6 +5,11 @@ const livestockTypes = {
   sheep: 'sheep'
 }
 
+const mappedClaimType = {
```

Comment:
I didnt like having to create this, given the object below, but this is IMO how it should have been implemented in the first instance.